### ISSUE TITLE
Implement theme classifier

### ISSUE THREAD

**Issue by jurrian** [openstate/open-raadsinformatie#166]
VNG and Open State have built a theme classifier which should be integrated in V1.

**Comment by jorritb** [comment]
This can be implemented as an enricher.

Sample output from the classifier:

```
{
 "https://id.openraadsinformatie.nl/71": {
   "Bestuur en Ondersteuning": 0.8579465049769507,
   "Economie": 0.6597606524237575,
   "Onderwijs": 0.8419365150444937,
   "Sociaal Domein": 0.9520022326249975,
   "Sport, Cultuur en Recreatie": 0.7100463811444125,
   "Veiligheid": 0.21210134016918814,
   "Verkeer, Vervoer en Waterstaat": 0.6319052977234323,
   "Volksgezondheid en Milieu": 0.12634399875623104,
   "Volkshuisvesting en Ruimtelijke Ordening": 0.8896927426069788
 }
}
```
It's currently not entirely clear how this information should be integrated and with what level of detail. If it has to be added to the model item itself a suitable ontology needs to be found first.

**Comment by tomkunzler** [comment]
Breyten is on holiday for the next three weeks. I would probably choose to list the top 3 themes without scores if they are higher then a specific level. People want a reliable score, so if a document scores on a theme but very low, do not show it. 

Option 1: Selecting a thema after I search from the menu bar
So for me there would be two options. I search for the word 'bike' and I only want documents that are about 'social affairs' so I click that theme in the menu bar which shows a selection of all possible themes. After I click 'social affairs' it only shows document that are classified as such with a high rating.

Option 2: Show themes with documents
It would also be nice to see a selection of max 3 themes after you have searched on something with the documents. So for example you find a 'Raadsvoorstel Fietsbeleid 2019' it shows behind the title the theme 'Infrastructure'.

Let me know if this works or we can have a call on clarification.

**Comment by jorritb** [comment]
I am going to work on this this week and hopefully have it ready for release next monday (september 2).

**Comment by jorritb** [comment]
I haven't been able to find a suitable ontology yet, so I [asked on Open Data SE](https://opendata.stackexchange.com/questions/15646/ontology-for-theme-of-a-text).

**Comment by joepio** [comment]
I suggest creating a new vocabulary, so creating new props, since this theme system is quite specific and consists of a domain-specific list of words.

Perpaps use the ORI:Meeting ontology for the theme predicate (ori:tags), and create a new set of resources for government themes.

**Comment by jurrian** [comment]
The theme classifier docker image has been deployed to the ORI cloud and can now be used. Next thing is building an enricher that queries the classifier endpoint.

**Comment by tomkunzler** [comment]
What is the latest progress/planning on this topic?

**Comment by joepio** [comment]
The PR is open, so it will be merged soon!

**Comment by jurrian** [comment]
PR has been merged, is in production now with v1.4.0
